### ISSUE TITLE
SteamVR Beta Input Action Issue

### ISSUE THREAD

**Issue by Mahdniss** [ValveSoftware/steamvr_unity_plugin#114]
Hello, I have created a new input Action via the SteamVR Input window under the default action set called "Move". I then mapped the action to the trackpad's touch via the UI after generating the input.

In a player movement script, I have created a public SteamVR_Action_Vector2 variable called move. I have assigned this variable via the editor's dropdown selection as \actions\default\in\Move.

I have also assigned a public Hand variable as my Camera Rig's LeftHand object.

In the update function of my player movement script, I call move.GetAxis(hand.handType) to get the axis of the touchpad.

The problem is, this error which seems to be specific to the input action itself: GetAnalogActionData error (/actions/default/in/Move): InvalidHandle handle: 1153002524869919043 UnityEngine.Debug:LogError(Object) Valve.VR.SteamVR_Action_Vector2:UpdateValue(SteamVR_Input_Sources) (at Assets/SteamVR/Input/SteamVR_Action_Vector2.cs:49) Valve.VR.SteamVR_Input:UpdateNonVisualActions(SteamVR_Input_Sources) (at Assets/SteamVR/Input/SteamVR_Input.cs:536) Valve.VR.SteamVR_Input:UpdateNonVisualActions() (at Assets/SteamVR/Input/SteamVR_Input.cs:508) Valve.VR.SteamVR_Input:Update() (at Assets/SteamVR/Input/SteamVR_Input.cs:322) Valve.VR.SteamVR_Behaviour:Update() (at Assets/SteamVR/Scripts/SteamVR_Behaviour.cs:178)

I am thinking that there is something incomplete about the custom input action, but I do not know what. Looking at the Live Input window shows that the action is not being called when touching the trackpad. I have tested both controllers to be safe.

**Comment by Mahdniss** [comment]
This was a misleading issue. I think during testing I created the wrong type of binding, and assigned it to "Touch" on the touchpad. I then created a Vector2 binding of the same name. The UI reported the binding was assigned to Touch, but the error would occur. After renaming the binding, and looking through the UI, I found that the Vector2 type must be bound to Position, and did not show up as an option under Touch. 

Assigning the binding to Position solved my problem. I think this could be an issue for others, and maybe a front-end quality-check issue with the UI, so I'll leave it open for now.
